src/Network/Network/ChangeLog.md

```diff
@@ -24,6 +24,7 @@
     - Added `TLSv1_3` to the validation set of `MinProtocolVersion`
     - Removed validation for null or empty cipher suites list since there can be empty cipher suites list for min protocol version of tls1.3
 * [Breaking Change] Changed default value of `-PrivateEndpointNetworkPoliciesFlag` to `Disabled` in `Add-AzVirtualNetworkSubnetConfig` and `New-AzVirtualNetworkSubnetConfig`
+* Onboarded Device Update for IoT Hub to Private Link Common Cmdlets
```

Comment:
Please move this on the bottom of the title of `## Upcoming Release` so that this msg can be add into latest release.

Reply:
I do not understand this comment. The line is already under the 'Upcoming Release'. Please provide more details - e.g., line number where this line should be moved to.

Reply:
Please move this  line between L21 and L22

Reply:
No, I am not going to do that. It does not make sense to me.

What you are saying is that, in the release notes, instead of 

- Change 1
- Change 2
- Change 3

You want to see

- Change 3
- Change 1
- Change 2

@MikhailTryakhov, maybe I am wrong, can you please break the tie here?

Reply:
The reason why I ask this request is that your change log will be put under version 4.18.0 instead of upcoming release section after your PR merges if you add change log in current position. 
![image](https://user-images.githubusercontent.com/19869716/177889975-1beb9aee-2f0b-4429-9b85-f4381df6de55.png)

Azure PowerShell team decides whether releases a new version for module according to is there any content under upcoming release section.

Reply:
That explains the confusion on my part.

This is what I see - why is that? I have no idea where your 4.18 is coming from.

![image](https://user-images.githubusercontent.com/61987922/177890832-31df7a2c-8368-42e8-8050-7dd6f992463a.png)

Reply:
This is terrible. You guys deal with this daily?

![image](https://user-images.githubusercontent.com/61987922/177891337-772c3212-4ec6-4a45-8e14-897faa411202.png)

Reply:
Please pull latest code from main branch.

Reply:
> This is terrible. You guys deal with this daily?

I would recommend following steps.

1. git remote --add upstream https://github.com/Azure/azure-powershell.git
2. git pull upstream main
3. {resolve conflicts}
4. git add .
5. git commit
6. git push